eval.py

```diff
@@ -18,26 +18,31 @@
 torch._inductor.config.triton.cudagraphs = True
 torch._dynamo.config.cache_size_limit = 100000
 
-# support running without installing as a package
-wd = Path(__file__).parent.parent.resolve()
-sys.path.append(str(wd))
-
-# hacky path setup for lm-evaluation-harness
-import os
-import sys
-
 from sentencepiece import SentencePieceProcessor
 
 from model import Transformer
 
-lm_evaluation_harness_path = '/'.join(
-    os.getcwd().split('/')[:-1] + ['lm-evaluation-harness'])
-sys.path.insert(0, lm_evaluation_harness_path)
-import lm_eval
-import main as lm_evaluation_harness_main
+try:
+    import lm_eval
+except:
+    lm_eval = False
```

Comment:
maybe rename `lm_eval` to something else to be clearer

Reply:
ok